compile error on msys2 with MS Windows7

Msys2＆Ｗindows7 で make してみたところ、コンパイルエラーが出ました。
sperl_yacc.h を生成する前にsperl_yacc_util.cをコンパイルするのがダメなようです。


$ make
gcc -MD -MF objs/sperl_switch_info.Po -std=c99 -g -O -Wall -Wextra -c -o objs/sperl_switch_info.o sperl_switch_info.c
gcc -MD -MF objs/sperl_op.Po -std=c99 -g -O -Wall -Wextra -c -o objs/sperl_op.o sperl_op.c
sperl_op.c: In function 'SPerl_OP_resolve_field_name':
sperl_op.c:651:17: warning: unused variable 'parser' [-Wunused-variable]
   SPerl_PARSER* parser = sperl->parser;
                 ^~~~~~
gcc -MD -MF objs/sperl_sub.Po -std=c99 -g -O -Wall -Wextra -c -o objs/sperl_sub.o sperl_sub.c
gcc -MD -MF objs/sperl_vm_stack.Po -std=c99 -g -O -Wall -Wextra -c -o objs/sperl_vm_stack.o sperl_vm_stack.c
gcc -MD -MF objs/sperl_bytecode_builder.Po -std=c99 -g -O -Wall -Wextra -c -o objs/sperl_bytecode_builder.o sperl_bytecode_builder.c
gcc -MD -MF objs/sperl_memory_pool.Po -std=c99 -g -O -Wall -Wextra -c -o objs/sperl_memory_pool.o sperl_memory_pool.c
gcc -MD -MF objs/sperl_constant_pool.Po -std=c99 -g -O -Wall -Wextra -c -o objs/sperl_constant_pool.o sperl_constant_pool.c
gcc -MD -MF objs/sperl_vm.Po -std=c99 -g -O -Wall -Wextra -c -o objs/sperl_vm.o sperl_vm.c
gcc -MD -MF objs/sperl_yacc_util.Po -std=c99 -g -O -Wall -Wextra -c -o objs/sperl_yacc_util.o sperl_yacc_util.c
sperl_yacc_util.c:10:24: fatal error: sperl_yacc.h: No such file or directory
 #include "sperl_yacc.h"
                        ^
compilation terminated.
make: *** [Makefile:15: objs/sperl_yacc_util.o] Error 1

--- a/Makefile
+++ b/Makefile
@@ -25,7 +25,7 @@ sperl_yacc.c: sperl_yacc.y
 	mv -f sperl_yacc.tab.h $(@:%.c=%.h)
 sperl_yacc.h: sperl_yacc.c
 $(OBJDIR)/sperl_toke.o: sperl_toke.c sperl_yacc.h
-$(OBJDIR)/sperl_yacc.o: sperl_yacc.c sperl_yacc.h
+$(OBJDIR)/sperl_yacc_util.o: sperl_yacc_util.c sperl_yacc.h
 libsperl.a: $(libsperl_OBJS)
 	$(AR) crs $@ $(libsperl_OBJS)
 all: libsperl.a

